_episodes/01-short-introduction-to-Python.md

```diff
@@ -70,11 +69,11 @@ Hello World
 ~~~
 {: .output}
 
-## Introduction to Python built-in data types
+## Introducción a los tipos de datos incorporados de Python
 
 ### Strings, integers, and floats
 
-One of the most basic things we can do in Python is assign values to variables:
+Una de las cosas más básicas que podemos hacer en Python es asignar valores a las variables:
 
 ~~~
 text = "Data Carpentry"  # An example of a string
```

Comment:
Sugerencia: traducir el comentario  An example of a String  como  Un ejemplo de un **string** (cadena de caracteres)

Reply:
Gracias, Monica, en las convenciones decidimos dejar los tipos de datos sin traducir

Reply:
Pero si quieres agregar la traduccion en parentesis me parece bien, por favor manda un PR